src/components/modules/ui.ts

```diff
@@ -591,9 +591,7 @@ export default class UI extends Module<UINodes> {
     /**
      * Clear Selection if user clicked somewhere
      */
-    if (!this.Editor.CrossBlockSelection.isCrossBlockSelectionStarted) {
-      this.Editor.BlockSelection.clearSelection(event);
-    }
+    this.Editor.BlockSelection.clearSelection(event);
```

Comment:
why the statement was removed?

Reply:
Because I changed `click` to `mousedown` above. On mouse down we are sure that selection should be remove. On click we are not because it might be added on the same click